.changeset/loud-apricots-fix.md

```diff
@@ -0,0 +1,10 @@
+---
+'example-app': patch
+'@backstage/create-app': patch
+'@backstage/plugin-scaffolder': patch
+---
+
```

Comment:
Probably want to add a little more example to the changeset here as it's a breaking change. And example with the `bindRoutes` thing is probably good enough :)

Reply:
I have added some more context and an example for what needs to be done to keep the current functionality.